python/dolfinx/io/gmshio.py

```diff
@@ -55,6 +60,25 @@
 }
 
 
+class MeshData(typing.NamedTuple):
```

Comment:
See https://github.com/FEniCS/dolfinx/blob/cee28f63f67e3ec4b4e9f0e260e0dcd8a878d8dd/python/dolfinx/fem/function.py#L548 for an example of documenting a `NamedTuple` with Sphinx.